hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/AbstractAbfsIntegrationTest.java

```diff
@@ -25,6 +25,8 @@
 import java.util.UUID;
 import java.util.concurrent.Callable;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
import order

Reply:
Had assumed it's to be grouped with third-party imports; shifted to apache section